canonical-websites/www.ubuntu.com
# Use canonicalwebteam.django_views.TemplateFinder

This is to we can get Markdown support.

QA
--

First off, extract some Markdown files into your templates directory:

``` bash
wget -qO- https://transfer.sh/P0QS5/markdown.tgz | tar xvz -C templates
```

Now run the site:

``` bash
./run
```

Check the basic site works at http://0.0.0.0:8001/. Then go to http://0.0.0.0:8001/markdown to see markdown parsing in action. Inspect the markdown templates in `templates/markdown/` to check they make sense to you.
